src/main/java/com/laa/model/crime/CrimeAssessmentResult.java

```diff
@@ -0,0 +1,6 @@
+package com.laa.model.crime;
+
+public enum CrimeAssessmentResult {
+
+    PASSED, PASSED_WITH_CONTRIBUTION, FAILED, PASSED_WITHOUT_CONTRIBUTION, INELIGIBLE
```

Comment:
What does Failed mean in this context?  Im assuming if something goes wrong we will throw an exception

Reply:
PASSED and FAILED  for megs court while other 3 are crown courts resuts